### ISSUE TITLE
Issue about test code

### ISSUE THREAD

**Issue by sunshangquan** [jeya-maria-jose/TransWeather#2]
Dear Authors,

Thanks for you excellent work! I have a question about the test code:

In utils.py, you use the following codes to transform image from (1,3,480,720) to (480, 720, 3). 

im1 = im1[0].view(im1.shape[2],im1.shape[3],3).detach().cpu().numpy()
im2 = im2[0].view(im2.shape[2],im2.shape[3],3).detach().cpu().numpy()

But why not use “transpose” or “permute” to do that, for example:

im1 = im1[0].permute(1,2,0).detach().cpu().numpy()
im2 = im2[0].permute(1,2,0).detach().cpu().numpy()

I am not sure whether "view“ can be used to change the order of dimensions. If I change these lines, the PSNR drops by about 0.6dB. Am I wrong about this?

Thank you for your attention and looking forward to your reply.

**Comment by jeya-maria-jose** [comment]
I believe you are right about this. It does make sense to use permute instead of view. Will change the code and also the reported value in the next version. Thank you for pointing it out.
